backends/bmv2/bmv2stf.py

```diff
@@ -452,7 +452,7 @@ def parse_table_set_default(self, cmd):
             actionArgs.set(k, v)
         command = "table_set_default " + tableName + " " + actionName
         if actionArgs.size():
-            command += " => " + str(actionArgs)
+            command += " " + str(actionArgs)
```

Comment:
There was a bug in the STF test framework. Default action overrides were not using the correct syntax.